marc_to_solr/lib/traject_config.rb

```diff
@@ -1155,57 +1161,101 @@
 end
 
 # Process location code once
-# 852|b and 852|c
+# AVA|b and AVA|c
 each_record do |record, context|
```

Comment:
Same here. Please reverse the changes. The location_code is a compination of 852|b and 852|c